docs/io/unified.rst

```diff
@@ -911,6 +912,15 @@ By default, ``serialize_method`` for Time columns is equal to
 
 .. _table_io_hdf5:
 
+QDP
```

Comment:
It does not get a separate section here since it is "just" another ASCII format.

Reply:
Done